### PR TITLE
memory fixed for pod eviction

### PR DESCRIPTION
**Pull request by Pamir**
- Memory request/limit should be equal for pod priority
- JAVA_OPTIONS example added

### CODE REVIEW

File: charts/dependency-track/values.yaml [hunk 1]
```diff
@@ -64,8 +64,7 @@ resources:
     cpu: 1500m
     memory: 4Gi
   limits:
-    cpu: 4
```
Comment (davidkarlsen) [inline charts/dependency-track/values.yaml]: The limits there match dependency-track requirements (and is just a guideline). Why should it be changed?

**Comment by Pamir** [comment]
Hi,
According to kubernetes documentation. Memory request and limit should be equal.

https://cloud.google.com/blog/products/gcp/kubernetes-best-practices-resource-requests-and-limits

In dependency-track helm chart we should give -Xmx4g memory and 4.5 Memory Request/Limit

Kubernetes will restart pods in this order. It is related with pod priority. 

- No Request/Limit
- Running pods more than its limits
- Between Request and Limits


If we give request memory and memory limit equal scheduler will not restart dependency-track pod. 
Dependency track at least needs -Xmx4GB of memory and limit is 4GB. For native memory side it is a good practice to limit of  pods memory (1.2 x Heap size of java app) which is 4.8.


Page 41: https://www.slideshare.net/try_except_/optimizing-kubernetes-resource-requestslimits-for-costefficiency-and-latency-highload

File: charts/dependency-track/values.yaml [hunk 2]
```diff
@@ -74,6 +73,9 @@ tolerations: []
 affinity: {}
 
 env: []
+  ## Example: Configure JAVA_OPTIONS
```
Comment (davidkarlsen) [inline charts/dependency-track/values.yaml]: Note that JVM heap-size is calculated from limit: https://github.com/DependencyTrack/dependency-track/blob/master/src/main/docker/Dockerfile#L43

**Comment by davidkarlsen** [comment]
Please check the failing tests. You can see the output in the "Actions" tab

**Pull request closed without merging**